src/backend/tests/fixtures/get-test-server.ts

```diff
@@ -1,3 +1,10 @@
+import mockery from "mockery";
+import * as nodemailerMock from "nodemailer-mock";
+mockery.enable({
+  warnOnUnregistered: false,
+});
+mockery.registerMock("nodemailer", nodemailerMock);
```

Comment:
🐧 could move this into a sibling `mocks.ts` and `import {nodemailerMock} from "./mocks"` to make it slightly cleaner